unlock-protocol-com/src/components/pages/Grants/index.tsx

```diff
@@ -0,0 +1,358 @@
+import { Button } from '@unlock-protocol/ui'
+import { Link } from '../../helpers/Link'
+import { BsLaptop as LaptopIcon } from 'react-icons/bs'
+import { HiSpeakerphone as SpeakerPhoneIcon } from 'react-icons/hi'
+import { FaHandsHelping as HelpingIcon } from 'react-icons/fa'
+import { MdGeneratingTokens as TokenIcon } from 'react-icons/md'
+import { AiOutlineLink as LinkIcon } from 'react-icons/ai'
+import { UnlockIcon } from '../../icons/Unlock'
+
+const UNLOCK_GRANT_APPLICATION_LINK =
+  'https://share.hsforms.com/1gAdLgNOESNCWJ9bJxCUAMwbvg22'
+
+const UNLOCK_GRANT_PROGRAM_LINK =
+  'https://docs.unlock-protocol.com/unlock/governance/grants-bounties-and-matchings'
+
+const UNLOCK_GRANT_HELP = [
+  {
+    Icon: TokenIcon,
+    title: 'Get UDT token grant',
+    text: 'We will provide you a grant of 100 UDT tokens to kickstart your project on Unlock',
+  },
+  {
+    Icon: LaptopIcon,
+    title: 'Technical Guidance for Project Development',
+    text: 'Get help from our developers on discord. We will provide suggestions and prioritize your issues.',
+  },
+  {
+    Icon: SpeakerPhoneIcon,
+    title: 'Marketing & Promotional Support',
+    text: 'We will help you perfect your pitch, provide feedback, and promote you on our social handles with over thousands of followers each.',
+  },
+  {
+    Icon: HelpingIcon,
+    title: 'Connecting with community and investors.',
+    text: 'We will help you launch, connect with our community members so you can find early users for your product.',
+  },
+]
+
+const UNLOCK_GRANT_COMMITTEE = [
```

Comment:
for each of them let's add a link and point to their LinkedIn or Twitter?

Reply:
I don't have links to their some of their Twitter. Mind giving me a list?